lib/livebook_web/live/explore_live.ex

```diff
@@ -79,7 +79,7 @@ defmodule LivebookWeb.ExploreLive do
       </div>
     </div>
 
-    <.current_user_modal current_user={@current_user} />
+    <.current_user_modal current_user={@current_user} return_to={Routes.explore_path(@socket, :page)} />
```

Comment:
Why do we need to pass `return_to` all the way down and use `push_path`? The user modal is not tied to a URL and is closed with JS on submit.

Reply:
But if we want to send a flash to current LiveView, it only works if you use `push_redirect` or `push_patch`.
[`put_flash/3` docs](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#put_flash/3)

Reply:
Yeah, I didn't notice the flash before. Was there any specific reason for adding it?

Reply:
Me and @josevalim was talking about success feedbacks on Hubs form and he asked to also put it to Users form

Reply:
@aleDsz let's try to do it without push_redirect/push_patch. If it doesn't work, let's give up on adding flash to users right now. :)